src/components/Player/Pages/Trends/Trends.jsx

```diff
@@ -26,6 +30,38 @@ const Trend = ({ routeParams, columns, playerId, error, loading }) => (
       <TrendGraph
         columns={columns}
         name={strings[`heading_${routeParams.subInfo || trendNames[0]}`]}
+        tooltip={{
+          contents: (d) => {
+            const data = columns[d[0].index];
+            const trendStr = strings[`heading_${routeParams.subInfo || trendNames[0]}`];
+            return `<div class="${styles.tooltipWrapper}">
```

Comment:
I know that this might seem nasty, but the other option I see is `renderToString` which is unnecessary IMO.